src/MigrationTools/Services/TelemetryClientAdapter.cs

```diff
@@ -57,6 +68,27 @@ public void TrackEvent(string name, IDictionary<string, string> properties, IDic
         public void TrackException(Exception ex, IDictionary<string, string> properties, IDictionary<string, double> measurements)
         {
             _telemetryClient.TrackException(ex, properties, measurements);
+
+            var baseException = ex.GetBaseException();
+            var createMessage = new CreateMessage
+            {
+                DateTime = DateTime.UtcNow,
+                Detail = ex.ToString(),
+                Type = baseException.GetType().FullName,
+                Title = baseException.Message ?? "An error occurred",
+                Severity = "Error",
+                Source = baseException.Source,
+                User = Environment.UserName,
+                Hostname = System.Environment.GetEnvironmentVariable("COMPUTERNAME"),
+                Application = "Exception Visualizer",
+                ServerVariables = new List<Item>
+                    {
+                        new Item("User-Agent", $"X-ELMAHIO-APPLICATION; OS={Environment.OSVersion.Platform}; OSVERSION={Environment.OSVersion.Version}; ENGINE=VisualStudio"),
```

Comment:
Replace 'VisualStudio' with the engine running this. Azure DevOps or what ever is executing this.

Reply:
Done